.github/workflows/core-prepare-release.yml

```
@@ -9,7 +9,20 @@ on:
         type: string
 
 jobs:
+  Checks:
```

nit: could we call this something like
```suggestion
  Dependency-Validation:
```
or similar?